src/functions.ts

```diff
@@ -40,6 +41,19 @@ export const createPaymentMethod = async (
   }
 };
 
+export const createToken = async (): Promise<CreateTokenResult> => {
```

Comment:
Let's add some params in case we want to allow for different tokens in the future (e.g. PII persons etc)

`export const createToken = async ({ type: 'Card' | 'BankAccount' | 'PII' | 'Person', billingDetails: PaymentMethods.BillingDetails}): Promise<CreateTokenResult> => {`